src/annotateModelSEED.py

```diff
@@ -0,0 +1,330 @@
+# Porthmeus
```

Comment:
Please either rename file or functions to be consistent i.e. SEED or seed

Reply:
Done with current commit